webapp/src/modules/address/reducer.js

```diff
@@ -153,6 +161,97 @@ export function addressReducer(state = INITIAL_STATE, action) {
             }
           }
         }
+        case EDIT_ESTATE_PARCELS_SUCCESS: {
+          const { parcels, type } = transaction.payload
+          const user = state.data[transaction.from]
+          let updatedParcelIds = []
+
+          if (type === REMOVE_PARCELS) {
+            updatedParcelIds = [
+              ...user.parcel_ids,
+              ...parcels.map(p => buildCoordinate(p.x, p.y))
+            ]
+          } else {
+            updatedParcelIds = user.parcel_ids.filter(
+              parcelId =>
+                !parcels.some(
+                  estateParcel =>
+                    buildCoordinate(estateParcel.x, estateParcel.y) === parcelId
+                )
+            )
+          }
+
+          return {
+            ...state,
+            data: {
+              ...state.data,
+              [transaction.from]: {
+                ...user,
+                parcel_ids: updatedParcelIds
+              }
+            }
+          }
+        }
+        case TRANSFER_ESTATE_SUCCESS: {
+          const { estate } = transaction.payload
+          const user = state.data[transaction.from]
+          return {
+            ...state,
+            data: {
+              ...state.data,
+              [transaction.from]: {
+                ...user,
+                estate_ids: user.estate_ids.filter(
+                  estateId => estateId !== estate.asset_id
+                )
+              }
+            }
+          }
+        }
+        case DELETE_ESTATE_SUCCESS: {
```

Comment:
This is adding the parcels from the removed Estate to the address, but it's not removing the Estate (so after executing this case the address has both the Estate and the parcels). We need to remove the estate_id from this address's `estate_ids`

Reply:
We are supporting behind empty estates (the server returns them) but we are not showing them at front.